lib/tic_tac_toe.ex

```diff
@@ -11,24 +11,25 @@ defmodule TicTacToe do
     communicator.display(welcome)
 
     player_move = String.trim(communicator.read_input())
-    board = Board.place_a_symbol(board, player_move, "X")
-    communicator.display(communicator_formatter.format_board(board))
+    if InputValidation.is_input_numerical?(player_move) do
+      board = Board.place_a_symbol(board, player_move, "X")
```

Comment:
It looks like you're passing `player_move` as a `String.t()` here and then later calling `String.to_integer/1` in `place_a_symbol` to convert this to a usable value for playing the move. What do you think about using the return value of `Integer.parse/1` and having `place_a_symbol/3` take an integer instead of a string?

Reply:
I like this alot, will probably help with the validation later too, Ill change the check to use the `is_number` or `is_integer` for the numerical validation